src/it/hosting-maven_release_plugin_version-ok/invoker.properties

```diff
@@ -0,0 +1,21 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I copied it from another invoker.properties of that project. See the
'bad-bundle' one for example.
Le 24 déc. 2015 10:09 PM, "Daniel Beck" <notifications@github.com> a écrit :

> In src/it/hosting-maven_release_plugin_version-ok/invoker.properties
> <https://github.com/jenkinsci/maven-hpi-plugin/pull/26#discussion_r48426193>
> :
>
> > @@ -0,0 +1,21 @@
> > +#
> > +# Licensed to the Apache Software Foundation (ASF) under one
>
> *Licensed to the Apache Software Foundation (ASF)* -- double check where
> you're copying your license text from.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkinsci/maven-hpi-plugin/pull/26/files#r48426193>.
>